interview-questions.md

```diff
@@ -10,32 +10,32 @@ Type your initials here to confirm you read the instructions:
 
 1. What is the difference between a parameter and an argument?
 
-Your answer:
+Your answer: A parameter is a place holder variable and the argument is the value of the variable that is called
 
-Researched answer:
+Researched answer:A parameter is a place holder variable that is defined inside parentheses. An argument is the actual content passed to the function inside the parentheses of the function
 
 2. What is the difference between an object and a class?
 
-Your answer:
+Your answer: class is a blueprint and object is the ingridients an example can be a cooking recipe. 
 
-Researched answer:
+Researched answer: Classes are a a type of function they are the blueprint of objects. objects are a vairable that reference a class
 
 3. What is iteration?
 
-Your answer:
+Your answer: Iteration is increasing the value until the condition is met and example is i++ in a for loop
 
-Researched answer:
+Researched answer: execute code once for each item in a collection
```

Comment:
Definitely suggest elaborating some more on these interview questions. Think about it this way: you don't want your interviewer to doubt whether you know what you're talking about or not. So you want to provide as much explanation as you can.